model/user_test.go

```diff
@@ -15,6 +17,40 @@ var testToken = &types.OAuthToken{
 	Token: "123456",
 }
 
+func (ms *modelSuite) TestUserTokens(c *check.C) {
+	usr, err := ms.model.CreateUser("roz", testToken)
+	c.Assert(err, check.IsNil)
+	testTokenJSON, err2 := json.Marshal(testToken)
+	c.Assert(err2, check.Equals, nil)
```

Comment:
please use `check.IsNil` in all places, instead of `c.Assert(var, check.Equals, nil)`